substrate/frame/nomination-pools/src/lib.rs

```diff
@@ -2509,13 +2527,13 @@ pub mod pallet {
 		/// The dispatch origin of this call must be signed by the pool nominator or the pool
 		/// root role.
 		///
-		/// This directly forward the call to the staking pallet, on behalf of the pool bonded
+		/// This directly forwards the call to the staking pallet, on behalf of the pool bonded
```

Comment:
To keep things generic, since this pallet can interact with anything that implements `StakingInterface`, and is not directly dependent on `pallet-staking`.

```suggestion
		/// This directly forwards the call to an implementation of `StakingInterface` (e.g., `pallet-staking`) through [`T::StakeAdapter`], on behalf of the pool bonded.
```